packages/agreement-email-sender/src/index.ts

```diff
@@ -21,14 +22,8 @@ import {
 
 import { config } from "./config/config.js";
 
-const emailManager = initEmailManager({
-  smtpAddress: config.smtpAddress,
-  smtpPort: config.smtpPort,
-  smtpSecure: config.smtpSecure,
-  smtpUsername: config.smtpUsername,
-  smtpPassword: config.smtpPassword,
-});
-const pecEmailManager = initEmailManager({
+const sesEmailManager = initSESMailManager({ awsRegion: config.awsRegion });
```

Comment:
With this change, the `EmailManager` config is not used anymore. Consider removing it from the `config` and update the envs accordingly